target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -306,6 +306,10 @@ rp-n53)
 rt-ac51u)
 	set_wifi_led "$boardname:blue:wifi"
 	;;
+rt-ac54u)
```

Comment:
Since the introduction of device-tree based device discovery in 3eb839b5795e6f8d6461f94dbc4217c3641e7f19, the The "rt-ac54u" board identifier can be replace by the machine compatible from the dts file: "asus,rt-ac54u" .

This will also make the change in the target/linux/ramips/base-files/lib/ramips.sh unnecessary.

Reply:
other asus devices not add "asus,xxx"

I reject